### PR TITLE
New Plugin: Inro: Add support for PANTABOX

### PR DESCRIPTION
**Pull request by t-mon**
nymea-plugins-modbus pull request checklist:

- [x] Make sure the pull request's title is of format "Plugin name: Add support for xyz" or "New plugin: Plugin name"

- [x] Did you test the changes on hardware, if not (e.g. absence of required hardware), please mention a person to confirm it has been tested.

- [x] Did you update the plugin's README.md accordingly?

- [x] Did you update translations (`cd builddir && make lupdate`)?

### CODE REVIEW

File: inro/integrationplugininro.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                        },
+                        {
+                            "id": "db02bf36-1ce2-40b4-bf88-8a4e3842a63b",
+                            "name": "maxChargingCurrent",
+                            "displayName": "Maximum charging current",
+                            "displayNameAction": "Set maximum charging current",
+                            "type": "uint",
+                            "unit": "Ampere",
+                            "minValue": 6,
+                            "maxValue": 32,
```
[the remark this answers is not in the document]
Comment (t-mon) [reply inro/integrationplugininro.json:114]: Updated. On the first connect, the value will be limited to whatever the charger communicates what the max limit is.

File: inro/integrationplugininro.h [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+private:
+    PluginTimer *m_refreshTimer = nullptr;
+    QHash<Thing *, PantaboxModbusTcpConnection *> m_connections;
+    QHash<Thing *, NetworkDeviceMonitor *> m_monitors;
+
+    void setupConnection(ThingSetupInfo *info);
+};
+
+#endif // INTEGRATIONPLUGININRO_H
+
```
[the remark this answers is not in the document]
Comment (t-mon) [reply inro/integrationplugininro.h:66]: Only one. Which will be added by the IDE

File: inro/integrationplugininro.cpp [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+    Thing *thing = info->thing();
+    qCDebug(dcInro()) << "Setup thing" << thing << thing->params();
+
+    if (m_connections.contains(thing)) {
+        qCDebug(dcInro()) << "Reconfiguring existing thing" << thing->name();
+        m_connections.take(thing)->deleteLater();
+
+        if (m_monitors.contains(thing)) {
+            hardwareManager()->networkDeviceDiscovery()->unregisterMonitor(m_monitors.take(thing));
+        }
```
[the remark this answers is not in the document]
Comment (t-mon) [reply inro/integrationplugininro.cpp:93]: Both are valid and OK, depends what looks better and the mood of the day :-)

**Review by mhei — APPROVED** [review]
LGTM

**Pull request merged**